pycytominer/cyto_utils/cells.py

```diff
@@ -167,9 +184,12 @@ def set_subsample_random_state(self, random_state):
     def load_image(self):
         """Load image table from sqlite file"""
         # Extract image metadata
-        image_cols = "TableNumber, ImageNumber, {}".format(", ".join(self.strata))
+        image_cols = "TableNumber, ImageNumber, Metadata_Site, {}".format(", ".join(self.strata))
```

Comment:
I was hoping to eventually move this hardcoded `image_cols` into something that's defined upon class instantiation. Can you make this upgrade now? I think it should be fairly straightforward to move `image_cols` to be defined as `self.image_cols=["TableNumber", "ImageNumber", "Metadata_Site"]` in `SingleCells.__init__()`.

You'll then need to do some wrangling in `load_image()` to get it ready for sql ingestion.

Reply:
see #120

Reply:
I have moved `image_cols` to `SingleCells.__init__()`. It looked like there will be duplicate columns if `image_cols` and `strata` shared features (for e.g. `Metadata_Site`). So I have also removed duplicates using `np.union1d()`

Reply:
the `np.union1d()` is a beautiful idea.

Let's go one step further with defining `image_cols` in `SingleCells.__init__()`. How about instead of hardcoding them inside `__init__()` (which is one step better than hardcoding for sure), but we can also add them as a default argument upon class instantiation (two steps better!). In other words, 

```python
class SingleCells(object):
    """yada yada yada
    """
    def __init__(
        self,
        file_or_conn,
        ....,
        image_cols=["TableNumber", "ImageNumber", "Metadata_Site"]
    ):
    self.image_cols = image_cols
```

This will enable us to use any image column in future profiles. Delightfully devilish!

Reply:
Ah I see what you mean. Implemented here - https://github.com/cytomining/pycytominer/pull/129/commits/ba0c15ea782df5f103341a13c77b4ff239da1c8e